### ISSUE TITLE
question from a newbie

### ISSUE THREAD

**Issue by ngennaro** [hoverlabs/beam_particle#1]
I see the hover code as a library inside the Photon system but no Beam. Is it there to use and I'm just missing it?

**Comment by emranemran** [comment]
That file is part of the particle IDE. If you copy paste all the files into
the IDE, the example should work.

**Comment by ngennaro** [comment]
Thanks got it compiled. Need to get some jumpers to test as my kit are all female. 

Is there an example where you take the data from something like IFTTT or some other input as opposed to a static set of text?

**Comment by emranemran** [comment]
Yep, we have a few examples with ifttt and webhooks. We'll upload the
examples soon.

**Comment by emranemran** [comment]
We just uploaded an example to get weather + stocks via webhooks - check
out the /examples folder under beam_particle folder. If you have any
questions, shoot us an email at support@hoverlabs.co

Hope you enjoy playing with Beam!

**Comment by emranemran** [comment]
Oops almost forgot to provide our setup guide that goes with that example
code: http://www.hoverlabs.co/guide/morning-update

**Comment by ScruffR** [comment]
Can this issue be closed?
